blocks/job-openings/job-openings.js

```diff
@@ -1,190 +1,11 @@
-import { readBlockConfig } from '../../scripts/lib-franklin.js';
+const tagString = '<iframe src="//www.careers-page.com/netcentric" class="embed-job-openings"></iframe>';
+const range = document.createRange();
+const jobOpeningsBlock = document.querySelectorAll('div.job-openings');
+const jobOpeningsBlockItem = jobOpeningsBlock.item(0);
+const emptyDiv = jobOpeningsBlockItem.firstElementChild;
 
-let jobOpenings;
-let jobListOffset = 0;
-let selectedPosition = 'all';
-let selectedLocation = 'all';
+range.selectNode(jobOpeningsBlockItem);
+const documentFragment = range.createContextualFragment(tagString);
+jobOpeningsBlockItem.appendChild(documentFragment);
```

Comment:
typically we have used `innerHTML` or the DOM API in other places for adding HTML, any reason to do it via fragment here?

Reply:
After the discussion with @jonatan-lledo-netcentric , we went with this approach instead innerHTML.

Reply:
yes I recently learn about the fragment function and will be safer that use the innerHTML IMHO

Reply:
curious, what is safer about it?

Reply:
innerHTML is used to inject malicious code, so if there is no innerHTML, then there are also no injections at all. That's why

Reply:
but how does this prevent XSS? isn't HTML interpreted in the same way?